Formula/p/pawk.rb

```diff
@@ -0,0 +1,25 @@
+class Pawk < Formula
+  desc "Python line processor (like AWK)"
+  homepage "https://github.com/alecthomas/pawk"
+  url "https://github.com/alecthomas/pawk/archive/refs/tags/v0.8.0.tar.gz"
+  sha256 "ede09939bfd51c680fecd0203d56fd0632aea3407cbf2ff23b50fb683f1a0cea"
+  license "MIT"
+
+  uses_from_macos "python"
+
+  def install
+    inreplace "pawk.py", /\A.*/, "#!/usr/bin/env python3"
+    bin.install "pawk.py" => "pawk"
+  end
+
+  test do
+    (testpath/"elements.txt").write <<~EOS
+      # Symbol Name
+      H  Hydrogen
+      He Helium
+      Li Lithium
+    EOS
+    output = pipe_output("#{bin}/pawk -B 'd={}' -E 'json.dumps(d)' '!/^#/ d[f[1]] = f[0]' < elements.txt")
```

Comment:
pipe output is only needed if you're piping some ruby string into the command.
```suggestion
    output = shell_output("#{bin}/pawk -B 'd={}' -E 'json.dumps(d)' '!/^#/ d[f[1]] = f[0]' < elements.txt")
```

Reply:
Out of curiosity, why is the explicit `testpath` required inside the `do test` block? I see a lot of other formulae doing the same, like the [tinxml2](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/t/tinyxml2.rb) example from the documentation. But the documentation also explicitly states that &mdash;
>The [test do](https://rubydoc.brew.sh/Formula#test-class_method) block automatically creates and changes to a temporary directory which is deleted after run.

Reply:
`testpath` is just the reference to the test directory. It isn't explicitly required to make a new file like here, but it is useful to be explicit.